src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -401,17 +401,17 @@ staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods collectNested java
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods collectNested java.util.Collection groovy.lang.Closure
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods collectNested java.util.Collection java.util.Collection groovy.lang.Closure
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods combinations java.lang.Iterable groovy.lang.Closure
-staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains Object[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains boolean[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains byte[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains char[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains double[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains float[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains int[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains java.lang.Iterable java.lang.Object
+staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains java.lang.Object[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains long[] java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods contains short[] java.lang.Object
-staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods containsAll Collection[] java.lang.Object
+staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods containsAll java.lang.Iterable java.lang.Object[]
```

Comment:
Mistakes from #137 that I just noticed while I was here. Currently `StaticWhitelistTest.sanity` tolerates `ClassNotFoundException` since there are some entries corresponding to classes not accessible from the test classpath of this plugin.

Reply:
(though maybe these ought to be enumerated)

Reply:
Yes, I was thinking that maybe we could split the whitelist in two and strengthen the checks on the whitelist where everything is available, but it doesn't seem critical.

Reply:
> split the whitelist in two

Actually in four, since there is `generic-whitelist` + `jenkins-whitelist`. OTOH all the classes in `generic-whitelist` are resolvable except for one `groovy-cps` thing which has been obsolete for a long time and could probably just be deleted now.